#!/bin/sh /usr/share/dpatch/dpatch-run
## 14-logging.dpatch by Michael Meskes <meskes@debian.org>
##
## DP: Remove excessive logging.

@DPATCH@

diff -Naurp VirtualBox-1.6.4/src/recompiler/VBoxRecompiler.c.orig VirtualBox-1.6.4/src/recompiler/VBoxRecompiler.c
--- VirtualBox-1.6.4/src/recompiler/VBoxRecompiler.c.orig	2008-08-06 10:03:37.000000000 +0200
+++ VirtualBox-1.6.4/src/recompiler/VBoxRecompiler.c	2008-08-06 10:03:56.000000000 +0200
@@ -1521,7 +1521,7 @@ int remR3NotifyTrap(CPUState *env, uint3
         STAM_COUNTER_INC(&s_aStatTrap[uTrap]);
     }
 #endif
-    RTLogPrintf("remR3NotifyTrap: uTrap=%x error=%x next_eip=%VGv eip=%VGv cr2=%08x\n", uTrap, uErrorCode, pvNextEIP, env->eip, env->cr[2]);
+    /*RTLogPrintf("remR3NotifyTrap: uTrap=%x error=%x next_eip=%VGv eip=%VGv cr2=%08x\n", uTrap, uErrorCode, pvNextEIP, env->eip, env->cr[2]);*/
     if(   uTrap < 0x20
        && (env->cr[0] & X86_CR0_PE)
        && !(env->eflags & X86_EFL_VM))
